Kickstarter-iOS/Views/Cells/CommentCell.swift

```diff
@@ -109,6 +116,8 @@ final class CommentCell: UITableViewCell, ValueCell {
     self.flagButton.rac.hidden = self.viewModel.outputs.flagButtonIsHidden
     self.replyButton.rac.hidden = self.viewModel.outputs.replyButtonIsHidden
     self.viewRepliesStackView.rac.hidden = self.viewModel.outputs.viewRepliesStackViewIsHidden
+
+    self.postedButton.rac.hidden = self.viewModel.outputs.commentStatus.map { $0 != .retrySuccess }
```

Comment:
This comparison should be in the view model not in the cell.

Reply:
Thanks. I missed it too.